pages/verification/index.tsx

```diff
@@ -0,0 +1,169 @@
+                                            autoClose: 3000,
+                                            type: 'error',
+                                        })
+                                    }
+                                })
+                            },
+                            onError: (err) => {
+                                setIsLoading(false);
+                                console.error(err)
+                                toast('We failed to send your message', {
+                                    autoClose: 3000,
+                                    type: 'error',
+                                })
+                            }
+                        })
+                    }
+                },
+                onError: (err) => {
+                    setIsLoading(false);
+                    console.error(err)
+                    toast('humanID verification failed, please try again', {
+                        autoClose: 3000,
+                        type: 'error',
+                    })
+                }
+            })
+        }
+        if (props.isFailedVerify) {
+            toast('humanID verification failed, please try again', {
+                autoClose: 3000,
+                type: 'error',
+            })
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [])
+
+    return (
+        <BaseContainer className="bg-black">
+            <Helmet>
+                <title>Human ID Verification</title>
+            </Helmet>
+            <LayoutContainer>
+                <div className="h-full flex flex-col justify-between">
+                    <LoaderWrapper isLoading={isLoading}>
+                        <div className="flex-1-0-0 relative overflow-hidden rounded-2xl">
+                            <img className="h-full w-full absolute object-cover" alt="verification image" src="/image/Verification_Illustration.svg" />
+                        </div>
+                        <div className="pt-4 gap-y-4 flex flex-col h-min">
+                            <text className="font-inter font-medium text-2xl text-justify">To send this message, please verify that you’re not a bot.</text>
+                            <div className="border border-gray06 rounded-lg rounded-t-xl flex flex-col">
+                                <button className="shadow-3xl bg-humanId_blue" style={{
+                                    margin: '-1px',
+                                    backgroundImage: `url('/image/humanID_bg_button.svg')`,
+                                    backgroundSize: 'contain',
+                                    backgroundRepeat: 'no-repeat',
+                                    backgroundPosition: 'center',
+                                    height: '45px',
+                                    borderRadius: '8px'
+                                }} onClick={() => getLoginPage()} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Inline styles can make the code harder to maintain and understand. Consider moving these styles to a CSS class or a styled component.



<!-- This is an auto-generated comment by OSS CodeRabbit -->